wrangler.production.toml

```diff
@@ -0,0 +1,16 @@
+# wrangler.toml file for deploying to internal cloudflare zones/workers.dev instances.
+# this shouldn't be touched or used besides by the GitHub Actions workflow!
+
+name = "workers-docs"
+type = "webpack"
+account_id = "b54f07a6c269ecca2fa60f1ae4920c99" # cloudflaredocs.workers.dev
```

Comment:
Let’s make this its own env, something like `[env.migration-testing]` (or `[env.migration]`) and put that, plus the `[env.production]` stuff below, into the default `wrangler.toml`.